src/redis_client.ts

```diff
@@ -31,3 +35,15 @@ export type PromisifiedRedisClient = typeof redisClient & {
 };
 
 export default redisClient as PromisifiedRedisClient;
+
+const deduplicationRedisClient = redis.createClient(
+  (process.env.REDISCLOUD_URL_DEDUPLICATION ||
```

Comment:
Optionally -- it will default to the main Redis instance. For staging and production I've set this to point to a different [logical database](https://redis.io/commands/select) on the same Redis server. So you can access these keys by connecting to the staging/prod redis server and then running `SELECT 1` (and `SELECT 0 ` switches you back to the main database)

Reply:
Sounds great